pkg/operator/controller/ingress/deployment.go

```diff
@@ -385,11 +412,22 @@ func desiredRouterDeployment(ci *operatorv1.IngressController, ingressController
 
 		env = append(env,
 			corev1.EnvVar{Name: "ROUTER_SYSLOG_ADDRESS", Value: socketPath},
-			corev1.EnvVar{Name: "ROUTER_LOG_LEVEL", Value: level},
+			corev1.EnvVar{Name: "ROUTER_LOG_LEVEL", Value: accessLogLevel},
 		)
 		volumes = append(volumes, rsyslogConfigVolume, rsyslogSocketVolume)
 		routerVolumeMounts = append(routerVolumeMounts, rsyslogSocketVolumeMount)
 		deployment.Spec.Template.Spec.Containers = append(deployment.Spec.Template.Spec.Containers, syslogContainer)
+	case useSyslogEndpoint:
+		if len(syslogFacility) > 0 {
+			env = append(env, corev1.EnvVar{Name: "ROUTER_LOG_FACILITY", Value: syslogFacility})
+		}
+		env = append(env,
+			corev1.EnvVar{Name: "ROUTER_LOG_LEVEL", Value: accessLogLevel},
+			corev1.EnvVar{Name: "ROUTER_SYSLOG_ADDRESS", Value: syslogEndpoint},
+		)
+		if len(accessLogFormat) > 0 {
+			env = append(env, corev1.EnvVar{Name: "ROUTER_SYSLOG_FORMAT", Value: accessLogFormat})
+		}
```

Comment:
Time to introduce some constants for the environment variables? Worth making a file just for them?

Reply:
I introduced constants for the new variables in `deployment.go`.  I'm not sure it would be helpful to put them in a separate file—they don't make much sense to use in other files.